crates/sage-cli/src/output.rs

```diff
@@ -19,7 +19,34 @@ impl FromParallelIterator<SageResults> for SageResults {
             .reduce(SageResults::default, |mut acc, x| {
                 acc.features.extend(x.features);
                 acc.quant.extend(x.quant);
-                acc.ms1.extend(x.ms1);
+                match (acc.ms1, x.ms1) {
```

Comment:
I think we can make this a "method" for the enum instead of being here, and repeated with the serial implementation.